### PR TITLE
Feature/ListItemBlockchain - DAO-636 

### PR DESCRIPTION
**Pull request by Fabricevladimir**
- added ListItemBlockchain with corresponding story and test
- fixed failing tests

### CODE REVIEW

File: packages/ui-components/src/components/listItem/blockchain.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        <IconRadioDefault width={20} height={20} className="text-ui-400" />
+      )}
+    </Container>
+  );
+};
+
+type SelectedProps = {
+  selected: boolean;
+};
+const Container = styled.div.attrs(({selected}: SelectedProps) => {
```
Comment (sepehr2github) [inline packages/ui-components/src/components/listItem/blockchain.tsx:46]: The card has a 12px border-radius. please add  'rounded-xl' :))

File: packages/ui-components/__tests__/cardProposal.test.tsx [hunk 2]
```diff
@@ -11,7 +11,33 @@ describe('cardProposal', () => {
   }
 
   test('should render without crashing', () => {
-    const element = setup({});
+    const element = setup(properties);
     expect(element).toBeInTheDocument;
   });
 });
+
+const properties = {
```
Comment (sepehr2github) [inline packages/ui-components/__tests__/cardProposal.test.tsx:19]: Thank you. Looks much cleaner! 👍

**Review by sepehr2github — APPROVED** [review]
Thank you!. Just a minor comment.

**Review by RakeshUP — APPROVED** [review]

File: packages/ui-components/src/components/listItem/blockchain.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/ui-components/src/components/listItem/blockchain.tsx:46]: Nice catch! Thanks.

**Pull request merged**